MqttInput.go

```diff
@@ -0,0 +1,60 @@
+package dnstap
+
+import (
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+)
+
```

Comment:
This helps clarify our intention that MqttInput implements this interface and provides a small amount of compile-time type checking.
```suggestion

var _ dnstap.Input = (*MqttInput)(nil)

```